scripts/build.sh

```diff
@@ -129,6 +129,8 @@ build_requested_canisters() {
         script_dir=$(dirname $0)
         project_dir=$(realpath "${script_dir}/..")
 
+        cp "$project_dir/src/integration-tests/evm_rpc.wasm.gz" "$WASM_DIR/evm_rpc.wasm.gz"
```

Comment:
Please remove the wasm from the repository and download it in the `get_dependencies.sh` script